nasa/earthdata-search
# EDSC-3890: EDSC shows empty granules for the collections list after hitting back to search on collections with customizable options

# Overview

### What is the feature?

Fixed bug where clicking Refine your search on a collection with too many granules loads 0 Granule collections with no names.

### What is the Solution?

Added updatePath boolean to the PortalLinkContainer that renders the "Refine your search" button so that we reload the page when navigating after clicking the button.

### What areas of the application does this impact?

Just EDSC ChunkedOrderModal.js

# Testing

### Reproduction steps

- **Environment for testing:** UAT
- **Collection to test with:** C1225776654-ASF

1. go to http://localhost:8080/search?ee=uat&q=C1225776654-ASF
2. Click on the first collection
3. Click Download All
4. Select the first Harmony access method.
5. Click Download Data
6. Click Refine your search
7. Observe the page loading all the collections again and that all URL params are maintained.

### Attachments
<img width="1609" alt="Screenshot 2023-11-29 at 3 53 42 PM" src="https://github.com/nasa/earthdata-search/assets/8591291/7c897e07-c57e-4553-942d-f46b2aedde41">
<img width="1609" alt="Screenshot 2023-11-29 at 3 53 48 PM" src="https://github.com/nasa/earthdata-search/assets/8591291/e9f3d087-ce0b-4807-b80d-86081c0a13ec">
<img width="1609" alt="Screenshot 2023-11-29 at 3 53 59 PM" src="https://github.com/nasa/earthdata-search/assets/8591291/4a15e0de-3714-4a72-a93a-c1120f5aea55">
<img width="1609" alt="Screenshot 2023-11-29 at 3 54 05 PM" src="https://github.com/nasa/earthdata-search/assets/8591291/527fcfd7-14c4-4282-aeea-203efe66009b">
<img width="1609" alt="Screenshot 2023-11-29 at 3 54 11 PM" src="https://github.com/nasa/earthdata-search/assets/8591291/1aa43543-3541-4c82-90ae-7cc2932a66bf">

# Checklist
- [x] I have added automated tests that prove my fix is effective or that my feature works (No tests to add, no new logic)
